### DISCUSSION TITLE
how do I backup my wallet?

### DISCUSSION THREAD

**Discussion by ralyodio** [shocknet/wallet2#206] [Q&A] [answered]
Is there a seedphrase or something? I don't see anything obvious.

Also can I swap between Bitcoin and LN? I don't know how to convert BTC to LN and back.

**Comment by shocknet-justin** [comment, marked as answer]
Howdy,

The seed phrase occurs either on the Lightning.Pub side when setting up a node on a remote system, or once a cloud node has provisioned automatically (this will come in the form of a notification after it has occurred). 

Logging in from the authentication menu will backup your wallet client keys to Nostr (we offer a relay service or you can use a nip07 extension to define your own)

You can generate a chain address from the receive screen and once that's confirmed it'll be spendable as sats, chain spends are not currently enabled however... hope to get to that soon:tm:
